### PR TITLE
fix: Typo in http/conditional_requests/index.md

### PR DESCRIPTION
**Pull request by pkchv**
### Description

Fixes minor typo in HTTP conditional requests docs. :nail_care:

### CODE REVIEW

File: files/en-us/web/http/conditional_requests/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ All conditional headers try to check if the resource stored on the server matche
 Comparing versions of the same resource is a bit tricky: depending on the context, there are two kinds of _equality checks_:
 
 - _Strong validation_ is used when byte to byte identity is expected, for example when resuming a download.
-- _Weak validation_ is used when the user-agent only needs to determine if the two resources have the same content. This is even if they are minor differences; like different ads, or a footer with a different date.
+- _Weak validation_ is used when the user-agent only needs to determine if the two resources have the same content. This is even if there are minor differences; like different ads, or a footer with a different date.
```
Comment (bsmth) [inline files/en-us/web/http/conditional_requests/index.md:30]: I think there's some more clarification we could add here while we're at it, what do you think?

```suggestion
- _Weak validation_ is used when the user-agent only needs to determine if two resources have the same content. The resources may be considered the same even if minor differences exist, such as different ads or a footer with a different date.
```

**Comment by bsmth** [comment]
Thank you! There's a suggestion above for another improvement while we're at it.

File: files/en-us/web/http/conditional_requests/index.md (hunk 1 above)
Comment (pkchv) [reply files/en-us/web/http/conditional_requests/index.md]: Looks great, added to the PR. Thank you.

Comment (bsmth) [reply files/en-us/web/http/conditional_requests/index.md]: Super, tnx

**Review by bsmth — APPROVED** [review]
Well done on your first PR here! If you're looking for other ways to contribute, we have a [contributor task board](https://github.com/orgs/mdn/projects/25). Issues labeled with `good first issue` are a great place to start.

Thank you :)

**Pull request merged**